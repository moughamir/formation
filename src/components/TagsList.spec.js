import React from 'react';
import TagsList from './TagsList';
import TestRenderer from 'react-test-renderer';

describe('TagList', () => {

  it('Should render as expected', () => {
    const tree = TestRenderer
      .create(<TagsList tags={[`css`, `html`, `python`]}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  })
})
