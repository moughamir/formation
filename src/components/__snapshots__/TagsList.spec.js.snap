// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagList Should render as expected 1`] = `
<div>
  <code>
    css
  </code>
  <code>
    html
  </code>
  <code>
    python
  </code>
</div>
`;
